### PR TITLE
torch.norm -> torch.nn.functional.normalize

### PR DESCRIPTION
**Pull request by pohanchi**
#166 sv downstream task `expert.py` ```torch.norm``` change to ```torch.nn.functinal.normalize``` add epilson.

### CODE REVIEW

**Comment by leo19941227** [comment]
太照了吧 @pohanchi ! 感謝爸爸!
